internal/command/deploy/deploy.go

```diff
@@ -121,11 +121,23 @@ var CommonFlags = flag.Set{
 	flag.StringSlice{
 		Name:        "regions",
 		Aliases:     []string{"only-regions"},
-		Description: "Deploy to machines only in these regions. Multiple regions can be specified with comma separated values or by providing the flag multiple times. --region iad,sea --regions syd will deploy to all three iad, sea, and syd regions. Applied before --exclude-regions. V2 machines platform only.",
```

Comment:
> --region iad,sea --regions syd will deploy to all three iad, sea, and syd regions.

I thought this part was kind of obvious. I can add it back if others find it helpful.

> Applied before --exclude-regions.

It isn't clear what this means. All the filters are always applied. Order shouldn't matter.

> V2 machines platform only

We shouldn't need language like this now that everything's on machines, right?